onnx/external_data_helper.py

```diff
@@ -8,6 +8,7 @@
 from itertools import chain
 from typing import Callable, Iterable, Optional
 
+import onnx.onnx_cpp2py_export.checker as C  # noqa: N812
```

Comment:
I would not import the checker module as C in external_data_helper.py. Instead consider

```suggestion
import onnx.onnx_cpp2py_export.checker as c_checker
```

or similar